modules/tensor_mechanics/src/userobjects/GeneralizedPlaneStrainUserObject.C

```diff
@@ -62,15 +67,15 @@ GeneralizedPlaneStrainUserObject::execute()
   for (unsigned int _qp = 0; _qp < _qrule->n_points(); _qp++)
```

Comment:
Just before this loop, let's add a call to a new function `getScalarVariableIndex()` that returns the index for the current element, and then use that to index into the residual.  In the base class, it would always return 0.